s30020-NikiforKoseda/zadanie5.txt

```diff
@@ -0,0 +1,382 @@
+# Copyright 2015 Tomasz Idzikowski
+# Przedmiot: UKOS
+# Skrypt: Wprowadzenie do podstaw PowerShell-a
+#
+# Żeby uruchomić jakikolwiek skrypt w PowerShell-u to domyślnie musi on być podpisany przez Microsoft
+# Ale można to przełączyć i w rzeczywistości praktycznie każdy tak robi bo inaczej nie da się używać PowerShell-a
+# Także, jak będziesz miał problem z uruchomieniem tego skryptu to uruchom powłokę PowerShell-a z uprawnieniami Admina
+# W pisz poniższe polecenie a następnie je zatwierdź (będzie dodatkowe pytanie o przełączenie)
+#
+# Poniższa linijka została już na stacjach w laboratorium wykonana. Nie należy jej uruchamiać w laboratorium.
+# Bo wymaga to uprawnień administratora.
+#
+# PS C:\Users\user> Set-ExecutionPolicy Bypass
+# Odpowiedz "T" lub "Y" (nie wiem jak to będzie wyglądać na polskim windowsie)
+#
+# Wtedy jak to będzie już przełączone to jako zwykły użytkownik będziesz mógł sobie uruchamiać swoje skrypty
+
+Szybkie przemyślenie - czy ustawienie czegoś takiego nie podnosi ryzyka na automatycznie odpalane internetowe skrypty mogące przejąć kotrolę nad komputerem, lub zebrać informacje, jak na memach z losowo odpalającym się okienkiem CMD?
```

Comment:
Słuszne spostrzeżenie. Natomiast przy rozpoczynaniu nauki z danym zagadnieniem staramy się nie utrudniach nadmiernie. Poza tym zanim uruchomimy skrypt należy się z nim zapoznać. Przeczytać i sprwdzić co wykona zanim to zrobi.

To jak jakby zanim odpalisz silnik samochodu musiałbyś poznać budowę silnika, jak działa ale z inżynierską precyzją co gdzie jak i dlaczego a dopiero potem jeśli byś uznał, że nie ma problemów to dopierow wtedy i tylko wtedy mógłbyś uruchomić silnik.

Bo skąd masz mieć pewność, że przekręcenie kluczyka nie spowoduje eksplozji?

Z resztą tak powinno się robić, że przed wyruszeniem w trasę robisz sprawdzenie stanu technicznego pojazdu. Tu podobnie.
Czy używał byś w kuchni płyty gazowej, o której wiesz, że jest niebezpieczna bo może doprowadzić do wybuchu bo wąż od gazu ma uszkodzoną uszczelkę? Przy czym zaznaczam: świadomie.
No nie. Tu także trzeba sprawdzić co uruchamiasz.
A jak ściągasz jakieś programy z sieci to najpierw traktujesz je antyvirusem czy uruchamiasz z mottem YOLO ?